<launch>
  <node pkg="ros_awsiot_agent" type="named_shadow.py" name="named_shadow" output="screen" >
    <param name="~endpoint" value="$(env AWSIOT_ENDPOINT)"/>
    <param name="~thing_name" value="$(env AWSIOT_THING_NAME)"/>
    <param name="~root_ca" value="/.aws/AmazonRootCA1.pem"/>
    <param name="~cert" value="/.aws/certificate.pem.crt"/>
    <param name="~key" value="/.aws/private.pem.key"/>
    <param name="~shadow_name" value="test"/>
    <param name="~publish_full_doc" value="True"/>
    <param name="~enable_upstream" value="True"/>
    <remap from="~input" to="upstream"/>
    <param name="~enable_downstream" value="True"/>
    <remap from="~output" to="downstream"/>
  </node>
</launch>